notify.js

```diff
@@ -125,6 +136,10 @@
             'icon' : this.options.icon
         });
 
+        for (var i in this.params) {
+            this.myNotify[i] = this.params[i];
```

Comment:
Any reason you've assigned these directly to `myNotify` instead of using a property? Could it be better accessed by something like `e.target.data.foo` etc?